src/engine/controls/enginecontrol.h

```diff
@@ -48,17 +48,17 @@ class EngineControl : public QObject {
             UserSettingsPointer pConfig);
     ~EngineControl() override;
 
-    // Called by EngineBuffer::process every latency period. See the above
-    // comments for information about guarantees that hold during this call. An
-    // EngineControl can perform any upkeep operations that are necessary during
-    // this call.
+    /// Called by EngineBuffer::process every latency period.
+    /// See the above comments for information about guarantees that hold during this call.
+    /// An EngineControl can perform any upkeep operations necessary here.
+    /// @param dRate current playback rate in audio frames per second
     virtual void process(const double dRate,
             mixxx::audio::FramePos currentPosition,
             const int iBufferSize);
```

Comment:
Not sure if there is more fitting type (did we already switch to `size_t` elsewhere?). Anyway, at least we don't use the type prefixes anymore, so this should be named `bufferSize`.

Reply:
I created #13819 for the not Macro related changes in this PR. Please follow-up there.